### PR TITLE
Added overall accuracy cell

### PR DESCRIPTION
**Pull request by NMO13**
Added overall accuracy cell to confusion matrix:

image

### CODE REVIEW

**Comment by thinkh** [comment]
@NMO13 Any idea why the x-axis in the detail plot is so cluttered? In [malevo-daily](https://malevo-daily.caleydoapp.org/) it is correct...

image

File: src/ConfusionMatrix.ts [hunk 1]
```diff
@@ -489,6 +497,19 @@ export class ConfusionMatrix implements IAppView {
       });
   }
 
+  renderOverallAccuracyCell(data: number[][], renderer: IMatrixRendererChain, labels: string[], singleEpochIndex: number[], colors: string[]) {
+    const maxVal = Math.max(...data.map((x: number[]) => Math.max(...x)));
```
Comment (thinkh) [inline src/ConfusionMatrix.ts]: Alternative to flatten the array first and use only one `Math.max`:
```ts
const maxVal = Math.max(...[].concat(...data));
```

File: src/ConfusionMatrix.ts [hunk 2]
```diff
@@ -489,6 +497,19 @@ export class ConfusionMatrix implements IAppView {
...
+
+    const res = {
```
Comment (thinkh) [inline src/ConfusionMatrix.ts]: Wrong code indentation

File: src/ConfusionMeasures.ts [hunk 3]
```diff
@@ -95,11 +95,35 @@ export function calcEvolution(matrices: NumberMatrix[], funct: (matrix: NumberMa
...
 }
 
+export function calcOverallAccuracy(matrices: NumberMatrix[]): number[] {
+  const order = matrices[0].order();
+  const arr = [];
+  if(matrices.length === 0) {
+    return arr;
+  }
+
+  for(const m of matrices) {
```
Comment (thinkh) [inline src/ConfusionMeasures.ts]: Why don't you use `map`?

```ts
return matrices.map((m) => calcSummedPercent1(m));
```

**Comment by NMO13** [comment]
> @NMO13 Any idea why the x-axis in the detail plot is so cluttered? In malevo-daily it is correct...

Because its epoch granularity dependent.... I fix that.

**Comment by thinkh** [comment]
@NMO13 Thanks. I saw now that it is already described as bug in #164. So you can work on it in a different branch.

**Pull request merged**